lib/helper.rb

```diff
@@ -0,0 +1,28 @@
+require 'json'
+
+module Helper
+
+  def load_json(file)
+    file = File.read(file)
+    JSON.parse(file)[0]
+  end  
+
+  def extractMenu(hash)
+    result = []
+    hash.each do |name,price| 
+      result << {:name => name, :price => price}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.